defaults/main.yml

```diff
@@ -4,3 +4,4 @@ new_relic_api_timeout: 10
 revision: ""
 changelog: ""
 description: ""
+validate_certs: "false"
```

Comment:
The default value _has_ to be `true`. 

It was `true` _before_ you introduced this change [because that is the default for `validate_certs`](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/uri_module.html#parameter-validate_certs) and therefor people that are using this expect it to be `true` by default.